platform/linuxbsd/wayland/wayland_thread.cpp

```diff
@@ -2424,8 +2424,7 @@ void WaylandThread::_wp_tablet_tool_on_frame(void *data, struct zwp_tablet_tool_
 		// According to the tablet proto spec, tilt is expressed in degrees relative
 		// to the Z axis of the tablet, so it shouldn't go over 90 degrees either way,
 		// I think. We'll clamp it just in case.
-		td.tilt.x = CLAMP(td.tilt.x, -90, 90);
-		td.tilt.y = CLAMP(td.tilt.x, -90, 90);
+		td.tilt = td.tilt.clamp(Vector2i(-90, -90), Vector2i(90, 90));
 
 		mm->set_tilt(td.tilt / 90);
```

Comment:
Realized this might not be good either, this does integer division, is this desired?
Should it be:
```suggestion
		mm->set_tilt(Vector2(td.tilt) / 90);
```